### PR TITLE
MOSIP-33024 : Pre-Reg Migration Java21

### PR DESCRIPTION
**Pull request by GOKULRAJ136**

### CODE REVIEW

File: pre-reg-113-115/pom.xml [hunk 1]
```diff
@@ -103,7 +138,15 @@
...
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-starter</artifactId>
-			<version>3.0.0</version>
+		</dependency>
+		<dependency>
+			<groupId>org.junit.vintage</groupId>
+			<artifactId>junit-vintage-engine</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>io.mosip.kernel</groupId>
```
Comment (dhanendra06) [inline pre-reg-113-115/pom.xml:147]: please comment auth-adpter-dependency

File: pre-reg-113-115/src/main/java/com/reencryptutility/ReEncryptUtilityApplication.java [hunk 2]
```diff
@@ -5,20 +5,27 @@
...
+import org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration;
 import org.springframework.context.ConfigurableApplicationContext;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.FilterType;
 import org.springframework.web.client.RestTemplate;
 
 @SpringBootApplication
-@ComponentScan(basePackages={ "io.mosip.commons.khazana.*" ,"com.reencryptutility.*"}, excludeFilters = {
+@ComponentScan(basePackages={ "io.mosip.commons.khazana.*" ,"com.reencryptutility.*", "io.mosip.*", "io.mosip.kernel.auth.defaultadapter"},
```
Comment (dhanendra06) [inline pre-reg-113-115/src/main/java/com/reencryptutility/ReEncryptUtilityApplication.java]: io.mosip.kernel.auth.defaultadapter scan using properties file

File: pre-reg-113-115/pom.xml (hunk 1 above)
Comment (GOKULRAJ136) [reply pre-reg-113-115/pom.xml:147]: commented the auth adapter dependency

File: pre-reg-113-115/src/main/java/com/reencryptutility/ReEncryptUtilityApplication.java (hunk 2 above)
Comment (GOKULRAJ136) [reply pre-reg-113-115/src/main/java/com/reencryptutility/ReEncryptUtilityApplication.java]: Added "io.mosip.kernel.auth.defaultadapter" in the properties file and updated the scan package accordingly.

**Review by dhanendra06 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**